terraform/live/rabbitmq/az/prometheus/terragrunt.hcl

```diff
@@ -0,0 +1,23 @@
+include "root" {
+  path = find_in_parent_folders()
+}
+
+include "k8s_providers" {
+  path = "${dirname(find_in_parent_folders())}/k8s_providers.hcl"
+}
+
+include "envcommon" {
+  path = "${dirname(find_in_parent_folders())}/../../_envcommon/prometheus.hcl"
+}
+
+
+dependency "aks" {
+  config_path = "../aks"
+  mock_outputs = {
+    client_key             = "LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBdzMySy9IWjBpc1hTdHd6QTNYT1Mzb1M4ZkZ6U1ppSzV6dnpwNEU0YUZyTkpIN0JVCnVnWDdvM0RnVHFnamtqd1BZNkdMQUhMcTV6Z2xIWWNMeGgrckxRbFdKMXVlbnlPUDB3TEVBSll2VXdWU1pINUkKVVU1UEZoajVOWUw0dXM0SE9LUUpPWDU1cmw2b0lRcHl2VVZXYUh1MCtBQXFwWGJ3NEZuSHk4UlpLTnk0WTZmegprTHFEQm43dlpLRUlSZXM0U1JZbUlBa1ZNVjF5TVV1TUdESS8rOHh2UW9tZHdWdGxqbktZYzVDM2hoNFVLNGd2CnhuWDZTNDNGMTdyMkZoS3VxZW9IRnllOGtpdXZ4VlErU3NNd1NrRFhlbS9yeHhOWDMzODBwSkdzWFNVa096Q2MKSm05MGdCV3VreWRidE5OMFJrQzU3VjY2UkFnbHdscEZBSHU0UXdJREFRQUJBb0lCQUNlUmRKMVBHSjJ5RnNsbwpNbTFnWkR6WDN4TEk4dWtyWkRLS0xJaHpKb0tydlVSRnU1Qi9mNFNBWjFaa1gvYk1ZMDBHT0kxWnRVUWhzQmwrCjZSb3I5TWpEMkFWVlhSRFc4TmlEK2Nvb20rR1E5VEZVQWkwNnpYVXFqdDZPNk90ZW9rV0xwalpWN0Nrb0RaM0sKY0pqd3ZPR2ZFRzBXRTR1MFBHTWNxUDBNZkFOeDlxendidXhQK09NU2x4K2t1TU14c1lpZTBZVEZ0anU2a0F5aQpTUWxXTEcwUmd1T20zWEwyeWhXb0QwS0MxR2ZjR2VnZGUzaUlxT3MrRExCZ3FXYWQ3dlQ5UXZzbXU2K3pOemk5Ckl6Z2hGbDFRVEZXejVkLy8yTlJDK3JCSXRJT1VxVmthc3BLMFlSRmhFWmVEY0FrakJxN1VDWVBpYVUvN0ZPT3UKNXg3dldpa0NnWUVBOFBOM3JhL2gxa2NXTVpHRkRtRWxTYjhZVzRkZmRUYmphK25sM0FoUDBRaVVOUDRHa3pXVgpySjQ1OHExSVJ1bElaZ0FKdjdGWW16MGNJN0FlenRxSjZOVDkwN0pndUpvemcxSjYyd2lqTGU2N0psWXlVcEJJCnltWVhqRkVyY2dFZHhTY1AzaEJKTXRKRko4OUZLVWZlYVhTYm9UYUN2OThSakNKVjhCS3FnUThDZ1lFQXo3TTIKS1dyVjZVU29yTXhxWVRpNzRkOTFuWmxrT2RjUkxZZHhudTZTRHRzc3Bpb25nU2tOeUptNGhoTWhKaC8yQXJIbgpQU2NBbG9FS21RNkR2cEczZS83Zno3bmtIeE5DalptbS84UnM0dk5RaksrWDhlQU1TYnVrZUNhUGRFNy9wemp6CkI1YWFtRUl1a2hBbS9LckZPUm03VW44cC8vSGxSYTNxaUd2dExZMENnWUVBaCtMR3U4MVZMK01uUVEwd1hNdXMKbThxSmtoVG1ZTlJ6clloYmVadTJaOHpPeUpQNllVcjIva0s5cTM3T0pDK3NCTnBBcFRyL0hZMTVNU2RDT0taRwo0R2crUGdWN1JxQ3dMSDkzaWM5MU41SkxPWE9qTGF6eFlEUHpVODd6RVhoUHNjQTEwcWFjb1d5MFU1OHhBZmJtCkY5TUxjZityL1Vsd08vZWhQNk5IUlVFQ2dZQkVQN0E1ZGRpcW1Sd1FBKytrMkw2c0h3Q0g4K0tDT3hCeTlJRzYKWnN4cHcwMXRvSVp3bzZ2YTBlQlhkaXU3V2w4OVRYWWRteFpQWGtHOFA0Kzg3Q2J0d0xlTWk1blJPRVB6UVowbAppRWI3WUZhRWxDS0tvVU9JYXZkRlgrNWRtcVBaTUJaejFPSWh6QVYvaTMwMEkyMUJNUnBqcGxPSHZkWEh5cms1CmNMVno5UUtCZ0NTYUxUZHNwK3BWSkMwSzNyREtab3l0Sis3bVQ1UEZpcXVSaDVBcnZFR243Sk1ncmFWMGFFZjYKM3JCd1FuYmRMdzJnNHJaTk5NN3l1V3VUdHJ2cTRTb2pNU3RnK0paOGlZSU5pMXhGbHlma1pTQXA5a1BCZ3lHRwp5bDRXdTZORDFTZ1VtdklQdlVqTW1QN0d5VVZqS1F5bUZjR3R1d3VWZjhUUWtXVHZTVlVZCi0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCg=="
+    cluster_ca_certificate = "dGVzdA=="
```

Comment:
Same as above.

Reply:
The kubectl or helm provider checks whether the client_key variable has the value for PEM format private key. So even the mock value needs to be in PEM format. The same goes for client_certificate variable. Only the cluster_ca_certificate can be a short random encoded string.

Reply:
It's really weird that it doesn't check format of `cluster_ca_certificate`.

Reply:
In AWS or GCP we also didn't have to create a valid mock certificate for `cluster_ca_certificate` variable so I guess it's just how the kubectl/helm provider works.